src/utils/router-utils.js

```diff
@@ -48,6 +48,12 @@ export async function scrollBehavior(to, from, savedPosition) {
     await this.app.$nextTick();
     return savedPosition;
   }
+  if (areEquivalentLocations(to, from)
```

Comment:
`areEquivalentLocations` does not detect hash changes, so say I move across hashes, like in the OnThisPageNav, wont this prevent the scrolling?

If so, then leaving the `to.meta.preventScrolling` on its own and leaving the `areEquivalentLocations` check down there is a better option.

Reply:
It does prevent it... Just tried